db/seeds.rb

```diff
@@ -81,7 +81,7 @@
                         brand: 'Never Summer',
                         image: File.open('app/assets/images/gnar_possum.jpg')
                       },
-                      { name: 'MACHETE GT SNOWBOARD',
+                      { name: 'MACHETE GT',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.